op-service/txmgr/txmgr.go

```diff
@@ -196,9 +245,15 @@ func (m *SimpleTxManager) Send(ctx context.Context, updateGasPrice UpdateGasPric
 				continue
 			}
 
-			// Submit and wait for the bumped traction to confirm.
+			// Increase the gas price & submit the new transaction
+			newTx, err := m.IncreaseGasPrice(ctx, tx)
+			if err != nil {
+				m.l.Error("Failed to increase the gas price for the tx", "err", err)
```

Comment:
In the error case we shouldn't try to resubmit the same tx?

Reply:
I lean to no. I guess it's technically fine for the global mempool, but I don't want us to spam the same transactions over & over.

Reply:
hmm, ok, gonna resubmit it